turbinia/workers/__init__.py

```diff
@@ -168,14 +168,19 @@ def close(self, task, success, status=None):
     self.status = status
 
     for evidence in self.evidence:
-      if evidence.source_path and os.path.exists(evidence.source_path):
-        self.saved_paths.append(evidence.source_path)
-        if not task.run_local and evidence.copyable:
-          task.output_manager.save_evidence(evidence, self)
+      if evidence.source_path:
+        if os.path.exists(evidence.source_path):
+          self.saved_paths.append(evidence.source_path)
+          if not task.run_local and evidence.copyable:
+            task.output_manager.save_evidence(evidence, self)
+        else:
+          self.log(
+              'Evidence {0:s} has missing file at source_path {1:s} so '
```

Comment:
Sorry, I didn't catch this before, but can you change this to `{1!s}` so that in case the `evidence.source_path` is None that python3 won't complain about type mismatches?

Reply:
done